v-next/hardhat-keystore/.eslintrc.cjs

```diff
@@ -0,0 +1,3 @@
+const { createConfig } = require("../../config-v-next/eslint.cjs");
+
+module.exports = createConfig(__filename, { onlyHardhatError: false });
```

Comment:
Why `onlyHardhatError: false`? I think it should be true.

Reply:
Done [here](https://github.com/NomicFoundation/hardhat/pull/5654/commits/232307dd3a2d64e8a943f810a13c981e5ac4ee3f)